README.md

```diff
@@ -17,11 +15,11 @@ You can find how to do this on our [support document](https://support.mycrypto.c
 * Python 2.7.X\*\*\*
```

Comment:
Do we still need Node and Python after removing the derivation path checker?

Reply:
Im going to leave this in for now until we do an analysis of our dependencies to make sure we don't use node-gyp anywhere, since that was what used python 2.7.x